doc-content/enterprise-only/openshift/customimage-smartrouter-proc.adoc

```diff
@@ -0,0 +1,146 @@
+[id='customimage-smartrouter-proc_{context}']
+:componentimage: smartrouter
+:component: Smart Router
+= Creating a custom Smart Router image with an additional JAR file to implement custom routing
+
+By default, Smart Router routes requests based on the container alias. If several {KIE_SERVERS} provide a service with the same container alias, Smart Router balances the load between them.
+
+In some cases, custom routing functionality is required. You can create a custom class to implement the custom routing and then create a custom Smart Router image with the class. You can push this image into your custom registry and then use it to deploy Smart Router.
+
+.Prerequisites
+
+* A JDK and Apache Maven are installed.
+* The project for deploying {PRODUCT} is created in your {OPENSHIFT} environment
+* You know the route for the {OPENSHIFT} image registry and have the permission to push images into the registry. For instructions about configuring the registry, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.5/html/registry/index[_Registry_] in {OPENSHIFT} product documentation.
+
+.Procedure
+
+. Download the `{PRODUCT_FILE}-container-sources.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
```

Comment:
I just addressed a DDF ticket about locating the zip file in the Sofware Downloads page on the Portal. I just clicked on all of the names in https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=rhpam&productChanged=yes and none of them were for the rhpam-7.9.0-container-sources.zip file.

So, I would confirm the link name and reword the sentence as in the following example: Note that I just used an example link. It is not the correct one.

Click Red Hat Process Automation Manager 7.8.0 Process Server for All Supported EE7 Containers to download the rhpam-7.9.0-container-sources.zip product deliverable file from the Software Downloads page of the Red Hat Customer Portal.

Reply:
I will review my links when CR1 is out, thanks! For now keeping for consistency with other docs